content/docs/3_reference/3_panel/1_blueprints/0_page/reference-article.txt

```diff
@@ -136,6 +136,54 @@ icon: page
 icon: 📚
 ```
 
+## Image options

```

Comment:
Let's create this as snippet, e.g. `snippets/docs/panel-field-image.php`, and then reference it in the various reference pages instead of having the same text there as copies. What do you think?

Reply:
I had originally intended to do that, but we would have to extend the docs tag to pass attributes, because the text is not exactly the same in all instances.

Reply:
Are there multiple differences? I only spotted one word - `pages`/`users`. If that would be the case, we can already do that. the `docs` tag already allows for a `field` attribute that we could use then.

Reply:
There are also the examples, e.g.


```yaml "site/blueprints/pages/note.yml"
image:
  back: "{{ page.myCustomBackColor }}"
```
vs

```yaml "site/blueprints/users/editor.yml"
image:
  back: "{{ user.myCustomBackColor }}"
```

Reply:
I missed that! Then let's leave it manual as it is right now.